addons/website_forum/models/forum.py

```diff
@@ -868,6 +902,14 @@ def _notify_record_by_inbox(self, message, recipients_data, msg_vals=False, **kw
             return
         return super(Post, self)._notify_record_by_inbox(message, recipients_data, msg_vals=msg_vals, **kwargs)
 
+    def go_to_website(self):
+        self.ensure_one()
+        return {
+            'type': 'ir.actions.act_url',
+            'target': 'self',
+            'url': '/forum/%s/question/%s' % (slug(self.forum_id), slug(self)),
```

Comment:
move it in _compute_website_url method of forum.post :)

Reply:
Faut donc que j'hérite de website.published.mixin mais après un forum ne se publie pas vraiment 
Then I have to inherit from website.published.mixin but a forum can't really be published.
So, I need to inherit from this model but only using one field ? (the _compute_website_url)

Reply:
non, tu peux avoir une methode privé sur n'importe quel module et l'appelé ou tu veux, peux importe les mixin que tu utilises.

En gardant le nmeme nom, ca reste cohérent quand tu cherche la fonction, et si un jour dans le futur on venait a jouter la mixin, bha la fontction existe deja et on aura pas besoin de la renommer.